code/modules/mob/living/silicon/ai/freelook/chunk.dm

```diff
@@ -114,7 +114,7 @@
 		var/turf/t = turf
 		if(obscuredTurfs[t])
 			if(!t.obscured)
-				t.obscured = image('icons/effects/cameravis.dmi', t, "black", 15)
+				t.obscured = image('icons/effects/cameravis.dmi', t, "black", 16)

```

Comment:
It would be better if these were using the defines. Maybe something like LIGHTING_LAYER + 1.